.github/workflows/ci.yml

```diff
@@ -236,11 +232,15 @@ jobs:
             which pip
 
       - name: Compilation Cache
-        uses: hendrikmuhs/ccache-action@v1.2
+        uses: actions/cache@v3
```

Comment:
I'm not sure this is equivalent. `ccache` works on a per-file basis I think, where as this doesn't seem to (at least as far as I can tell)

Reply:
Right, I reverted this change